### DISCUSSION TITLE
Jupyter Notebook Animations?

### DISCUSSION THREAD

**Discussion by HaydenRRamm** [Part-II-Computational-Physics/.github#10] [Q&A]
Been trying to get my animations to work in the Jupyter notebook using iPython HTML code libraries but I can't get them to work properly. Anyone managed to get their animations going in Google Colab?

**Comment by sebastianl2014** [comment]
Using VS code, my animations also not showing, even for the very first example
![image](https://user-images.githubusercontent.com/125830083/220039810-42245596-6a2e-4c91-8205-d2e77af20e68.png)

**Comment by NathanMagnan** [comment]
Hey!

Ah that's annoying...
In the notebook, have you tried both `%matplotlib notebook` and `%matplotlib ipyml` ?
In VS code, I suspect that you may not need either of these.

**Reply by HaydenRRamm** [reply]
Yep- tried both.

**Comment by HaydenRRamm** [comment]
I've tried %matplotlib ipympl and widget but I get this error: Warning: Cannot change to a different GUI toolkit: ipympl. Using notebook instead.

**Reply by NathanMagnan** [reply]
I've never had this error, but I would try to kill the current Jupyter kernel, then re-run the first cell

**Comment by HaydenRRamm** [comment]
ok, figured it out:

from matplotlib import rc

rc('animation', html='jshtml')

then I just call anim by writing "anim" at the end of the code and reducing the no. of frames so that Colab doesn't completely lose its mind
